### ISSUE TITLE
Paste and Match Style

### ISSUE THREAD

**Issue by aliebuck** [ttscoff/nv#271]
On OS X,  the keyboard shortcut Command + Shift + V should be used to Paste and Match Style, rather than Paste as New Note. It would be good to have Paste and Match Style available, and mapped correctly.

**Comment by andyinabox** [comment]
+1 on the original issue. From my experience this is a pretty standard across-the-board keyboard shortcut in OS X applications.

Additionally, I am frequently pasting in text from the web that maintains its styling -- even though I am editing in plain text. This often results in unwanted bolded text, which I can only remove by selecting "Fix text encoding..." and then re-saving the .txt file. In any other OS X application I would simply use Command-Shift-V to paste in as plain text. 

It would be nice if nvALT could maintain this standard OS X behavior, or even make the "Paste and Match Style" option available under a different keyboard shortcut. I'd rather not have to use an additional tool like Marker and Bullseye when I would only be using it with nvALT.

(Of course I realize that maintaining open source software is a lot of work, so thank you! This is just something that consistently gets in my way, and I don't yet have the skills to fix and PR myself)

**Comment by bunk3m** [comment]
I also get **bold** text each time I paste any text into nvALT.  This is frustrating since the original text isn't **bold** and I don't want it to be **bold**.  It means I have to select and cmd+b to remove the **bold**.  If the behaviour is that all text pasted into nvALT is supposed to be plain text, then that isn't what it is doing.

It seems like this behaviour has only started over the past months with the more recent updates. I don't remember this happening before.  (??But I'm probably mistaken??)

**Comment by ttscoff** [comment]
In Preferences->Storage->"Store and read notes on disk as," do you have 
it set to plain text files?

-Brett

**Comment by bunk3m** [comment]
No.  I was lazy and never changed the storage from the default database.

**Comment by ruuts** [comment]
@bunk3m issue was resolved for me when turning of "Copy basic styles from other apps".
